### DISCUSSION TITLE
请问下能否支持1.12.2

### DISCUSSION THREAD

**Discussion by ghost** [ELDEpendenci/ELDependenci#9] [Ideas]
其实现在1.12.2的服务器还是占主流的，希望能支持一下1.12.2版本

**Comment by eric2788** [comment]
恐怕不能，首先物品API在1.12.2和1.13.2之間改動很大，而且1.13.2之後版本也有幾個大改的
其次框架內有些API是在1.14版本以上才有的功能

不過當然本插件是開源的，你可以嘗試自己改改
